features/bevragen/persoon/gezag/gba/gezag-meerderjarige-gba.feature

```diff
@@ -0,0 +1,721 @@
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | V          | 000000048        |
+      En de persoon met burgerservicenummer '000000024' heeft een ouder '1' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000036                   |
+      En voor de persoon met burgerservicenummer '000000024' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000024       | V          | 000000048        |
+      En de persoon met burgerservicenummer '000000036' heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000048                   |
+      En de persoon heeft een 'kind' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000012                   |
+      En de persoon heeft een 'kind' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000024                   |
+      En de persoon heeft de volgende 'overlijden' gegevens
+      | naam                     | waarde   |
+      | datum overlijden (08.10) | 20231001 |
+      En de persoon met burgerservicenummer '000000036' heeft geen gezagsrelaties
+      En de persoon met burgerservicenummer '000000048' heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) |
+      | 000000036                   | 20120428                                                           |
+      En de 'partner' is gewijzigd naar de volgende gegevens
+      | burgerservicenummer (01.20) | datum ontbinding huwelijk/geregistreerd partnerschap (07.10) | reden ontbinding huwelijk/geregistreerd partnerschap (07.40) |
+      | 000000036                   | 20231001                                                     | O                                                            |
+      En de persoon met burgerservicenummer '000000048' heeft geen gezagsrelaties
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000048                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam                             | waarde    |
+      | type                             | Voogdij   |
+      | minderjarige.burgerservicenummer | 000000012 |
+      | derde.burgerservicenummer        | 000000048 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde    |
+      | type                             | Voogdij   |
+      | minderjarige.burgerservicenummer | 000000024 |
+      | derde.burgerservicenummer        | 000000048 |
```

Comment:
dit kan ik geloof ik bijna niet meer volgen, maar dit zijn de enige situaties dat de voogd ook echt een BSN heeft toch?
- heeft al voogdij over 12
- krijgt voogdij over 24 door overlijden partner.

Moet dit in 1 scenario worden gecombineerd?

Reply:
ik begrijp het onderscheid dat je ziet tussen 12 en 24 niet. De persoon heeft voogdij over beide kinderen van de partner. De situatie van kinderen 12 en 24 is toch identiek beschreven?

Reply:
daar ga je al.... :-)